app/src/main/java/com/asfoundation/wallet/billing/paypal/PayPalTopupViewModel.kt

```diff
@@ -70,6 +86,37 @@ class PayPalTopupViewModel @Inject constructor(
     )
   }
 
+  fun processPayPalResult(amount: String, currency: String) {
+    if (isFirstResultRun) {
```

Comment:
this seems a strange behavior. maybe the problem lies on another place for adding this logic...